query/src/test/scala/filodb/query/exec/AggrOverRangeVectorsSpec.scala

```diff
@@ -518,7 +525,31 @@ class AggrOverRangeVectorsSpec extends RawDataWindowingSpec with ScalaFutures {
     result(0).rows.map(_.getDouble(1)).toList shouldEqual counts
   }
 
-  it("should sum and compute max of histogram & max RVs") {
+  it ("should add NaN in topK") {
+
+    val samples: Array[RangeVector] = Array(
+      toRv(Seq((1000L,5.1), (2000L, 5.6d), (3000L, 4.0d))),
+      toRv(Seq((1000L,5.0), (2000L, 5.7d), (3000L, 4.4d)), ignoreKey2)
+    )
```

Comment:
Can you make sure this test case includes NaN in the beginning, NaN in the middle and NaN at the end of presented range vector?